data-otservbr-global/npc/alexander.lua

```diff
@@ -30,6 +30,24 @@ npcConfig.voices = {
 }
 
 local itemsTable = {
+	["exercise weapons"] = {
+		{ itemName = "durable exercise rod", clientId = 35283, buy = 945000, count = 1800 },
+		{ itemName = "durable exercise wand", clientId = 35284, buy = 945000, count = 1800 },
+		{ itemName = "exercise rod", clientId = 28556, buy = 262500, count = 500 },
+		{ itemName = "exercise wand", clientId = 28557, buy = 262500, count = 500 },
+		{ itemName = "lasting exercise rod", clientId = 35289, buy = 7560000, count = 14400 },
+		{ itemName = "lasting exercise wand", clientId = 35290, buy = 7560000, count = 14400 },
+	},
+	["creature products"] = {
+		{ itemName = "crystal ball", clientId = 3076, buy = 530, sell = 190 },
+		{ itemName = "life crystal", clientId = 3061, sell = 83 },
+		{ itemName = "mind stone", clientId = 3062, sell = 170 },
+		{ itemName = "Spellbook of Enlightenment", clientId = 8072, sell = 4000 },
```

Comment:
in some case you use "Spellbook of Enlightenment" and others you use "spellbook of enlightenment"
I recommend you to update all putting in lower case.

Reply:
This item should be in shields category.